### PR TITLE
Adds LimitedQuantity icon

### PR DESCRIPTION
**Pull request by kevzettler**
screen shot 2016-07-20 at 11 28 20 am

### CODE REVIEW

File: app/components/LimitedQuantity.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    };
+  },
+
+  render() {
+    const classes = classnames('tilt-icon tilt-icon-limited-quantity', this.props.className);
+
+    return (
+      <svg {...this.props} className={classes} viewBox="0 0 79 79">
+          <title>LimitedQuantity</title>
+          <g id="Page-1" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
```
Comment (vig9295) [inline app/components/LimitedQuantity.js]: Could we have all the styles in a stylesheet?

File: app/components/LimitedQuantity.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+              <g id="max_quantity_3">
```
Comment (vig9295) [inline app/components/LimitedQuantity.js]: We can get rid of all the unnecessary g tags.

File: app/components/LimitedQuantity.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                                  <path d="M47.1213203,25.5002096 L52.4305562,20.1909737 C52.8099136,19.8116163 52.815608,19.1734176 52.4254912,18.7833008 L51.7169088,18.0747184 C51.323548,17.6813575 50.6965553,17.6823339 50.3092358,18.0696534 L45,23.3788892 L39.6909642,18.0698534 C39.3116067,17.690496 38.673408,17.6848016 38.2832912,18.0749184 L37.5747088,18.7835008 C37.181348,19.1768616 37.1823243,19.8038542 37.5696438,20.1911737 L42.8786797,25.5002096 L37.5698438,30.8090454 C37.1904864,31.1884028 37.184792,31.8266016 37.5749088,32.2167184 L38.2834912,32.9253008 C38.676852,33.3186616 39.3038447,33.3176853 39.6911642,32.9303658 L45,27.6215299 L50.3090358,32.9305658 C50.6883933,33.3099232 51.326592,33.3156176 51.7167088,32.9255008 L52.4252912,32.2169184 C52.818652,31.8235575 52.8176757,31.1965649 52.4303562,30.8092454 L47.1213203,25.5002096 L47.1213203,25.5002096 Z M13.43,7.9 L23.7,43.45 L62.805,43.45 C64.385,43.45 65.965,42.66 66.36,41.08 L78.605,13.43 C79.79,10.665 77.815,7.9 75.05,7.9 L13.43,7.9 Z" id="Shape" fill="#009FDA"></path>
```
Comment (vig9295) [inline app/components/LimitedQuantity.js]: Could you remove the id and move the styles to the stylesheet?

**Comment by vig9295** [comment]
Need to update version on package.json

File: app/components/LimitedQuantity.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+const React = require('react');
+const classnames = require('classnames');
+
+module.exports = React.createClass({
```
Comment (tildedave) [inline app/components/LimitedQuantity.js]: this should use `defineSvg` to remove a lot of the boilerplate

File: app/components/LimitedQuantity.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    return (
+      <svg {...this.props} className={classes} viewBox="0 0 79 79">
+          <title>LimitedQuantity</title>
+          <g id="Page-1" stroke="none" strokeWidth="1" fill="none" fillRule="evenodd">
+              <g id="max_quantity_3">
+                  <g id="shopping_cart_empty">
+                      <g id="Page-1">
+                          <g id="shopping_cart_empty">
+                              <g id="Layer_1">
+                                  <circle id="Oval" fill="#009FDA" cx="63.2" cy="71.1" r="7.9"></circle>
```
Comment (tildedave) [inline app/components/LimitedQuantity.js]: should nuke all the `id` attributions

**Comment by vig9295** [comment]
The readme has some good instructions on the pattern we follow for the component file. You could look at some of the earlier commits too.

**Comment by msied** [comment]
+1

**Comment by kevzettler** [comment]
ping

File: app/stylesheets/LimitedQuantity.scss [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+.tilt-icon.tilt-icon-limited-quantity {
+  fill: $tilt-blue;
+  height: 30px;
+  width: 30px;
+
+  .tilt-icon-main {
+    stroke: none;
+    stroke-width: 1;
+    fillRule: evenodd;
```
Comment (BrettBukowski) [inline app/stylesheets/LimitedQuantity.scss]: should this be fill-rule?

**Comment by vig9295** [comment]
lgtm

**Pull request merged**